autoconnect/src/main.rs

```diff
@@ -83,6 +83,8 @@ async fn main() -> Result<()> {
         port, router_port
     );
 
+    actix_web::App::new();
+
```

Comment:
Leftover from something else?

```suggestion
```

Reply:
no idea where that came from and had to pull to see it. 

Should be gone now.